framework/branch-key-store.md

```diff
@@ -665,6 +848,13 @@ the keystore operation MUST call with a request constructed as follows:
 - `EncryptionContext` MUST be the [encryption context](#encryption-context) constructed above
 - `GrantTokens` MUST be this keystore's [grant tokens](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#grant_token).
 
+Every attribute except for `enc` on the AWS DDB response item
+MUST be authenticated in the decryption of `enc`.
+In Hierarchy Version `v1`, when AWS KMS Decrypt succeeds this attribute gets
+authenticated by KMS as they are included in the AWS KMS encryption context.
+For authentication of these attributes in Hierarchy Version `v2`,
+the operation MUST match the first 48 bytes of `Plaintext` returned by AWS KMS Decrypt operation with SHA-384 Digest for the beacon key of serialization of the [branch key context](#branch-key-context).
```

Comment:
this section talks about general branch key decryption, i think you may want to ermove beacon key here?